### PR TITLE
architecture: describe subkeys

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Adds a new section on subkeys and a note on subkeys with REE-FS TAs.

Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
I'll update the example output once we're happy with everything else.

**Comment by jforissier** [comment]
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Review by etienne-lms — COMMENTED** [review]
Minor comments.
`Acked-by: Etienne Carriere <etienne.carriere@linaro.org>`

File: architecture/subkeys.rst [hunk 1]
```diff
@@ -0,0 +1,204 @@
+.. _subkeys:
+
+#######
+Subkeys
+#######
+Subkeys is an OP-TEE-specific implementation to provide a public key
```
[the remark this answers is not in the document]
Comment (jenswi-linaro) [reply architecture/subkeys.rst:6]: Grammarly preferred the former.

**Comment by jbech-linaro** [comment]
Thanks @jenswi-linaro @etienne-lms and @jforissier !

**Pull request merged**